kvwrapper_etcd/kvwrapper_etcd.go

```diff
@@ -32,7 +43,11 @@ func (e EtcdWrapper) Set(key string, val string, ttl uint64) error {
 
 // GetVal returns a single KeyValue found at key
 func (e EtcdWrapper) GetVal(key string) (*kvwrapper.KeyValue, error) {
-	r, err := e.client.Get(key, false, true)
+	options := &etcd.GetOptions{
+		Sort:      false,
+		Recursive: true,
```

Comment:
Well, the old line is:

```go
r, err := e.client.Get(key, false, true)
```

and the function [is defined](https://github.com/coreos/go-etcd/blob/master/etcd/get.go#L10) as:

```go
func (c *Client) Get(key string, sort, recursive bool) (*Response, error)
```

so we had been doing `sort = false` and `recursive = true`, so I wanted to preserve that behavior if we're simply moving from one library to another.

Now maybe we don't need recursive; it does look like that, so maybe I should change it and test with that and make sure it still works.